config/default.yml

```diff
@@ -0,0 +1,79 @@
+blacklisted_items:
```

Comment:
Is yaml easier to access in python scripts? Does `default.yml` do some autoloading magic?

Reply:
We can use libraries like https://pypi.org/project/PyYAML/ to quickly parse yaml config files.

Reply:
Cool, thanks :)